rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -289,6 +289,82 @@ def db_set(self, table, record, *col_values):
             self.run("set", args=args)
 
     def create_client(self):
-        print "*********   call OvnNbctl.create_client"
+        print "*********   call OvsVsctl.create_client"
         client = self._OvsVsctl(self.credential)
         return client
+
+
+
+@configure("ovs-ofctl")
+class OvsOfctl(OvsClient):
+
+    class _OvsOfctl(object):
+
+        def __init__(self, credential):
+            self.ssh = get_ssh_from_credential(credential)
+            self.context = {}
+            self.batch_mode = False
+            self.sandbox = None
+            self.cmds = None
+
+        def enable_batch_mode(self, value=True):
+            self.batch_mode = bool(value)
+
+        def set_sandbox(self, sandbox, install_method="sandbox"):
+            self.sandbox = sandbox
+            self.install_method = install_method
+
+        def run(self, cmd, opts=[], args=[], stdout=sys.stdout, stderr=sys.stderr):
+            self.cmds = self.cmds or []
+
+            # TODO: tested with non batch_mode only for docker
```

Comment:
This is misleading. Please don't add docker related code if not tested. A TODO comment can be added for supporting docker in the future.

Reply:
Ok will do